usmqe/web/tendrl/mainpage/clusters/cluster_list/pages.py

```diff
@@ -44,23 +47,47 @@ def import_gluster_cluster(self, name=None, hosts=None):
                           for check only
                           TODO: not used for now
         """
+# TODO: Choose which cluster should be imported
+# TODO: Change cluster name
+#       https://github.com/Tendrl/api/issues/70
         self.start_import_cluster()
+        import time
```

Comment:
Why is this import in the middle of a function and not at the beginning of the file?